### PR TITLE
Delete plutus-playground

### PR DESCRIPTION
**Pull request by zeme-iohk**

### CODE REVIEW

**Comment by zeme-iohk** [comment]
Deleted all references to `plutus-playground*` and `web-ghc`
Still needs to refactor `playground-common`

File: README.adoc [hunk 1]
```diff
@@ -165,15 +164,6 @@ Run `cabal build plutus-pab` from the root to build the Plutus PAB library.
 
 See the link:./cabal.project[cabal project file] to see the other packages that you can build with `cabal`.
 
-=== Deployment
-
-The Plutus Playground is automatically deployed upon certain pushes to GitHub
-
-* https://plutus-playground-plutus-apps-staging.plutus.aws.iohkdev.io/[Staging] is deployed from every commit pushed to `main` (this URL subject to change)
```
[the remark this answers is not in the document]
Comment (zeme-iohk) [reply README.adoc:172]: From lines 35-40
```
=== User documentation

The main documentation is located https://plutus-apps.readthedocs.io/en/latest/[here].

The generated Haskell API documentation (haddocks) are here:
<https://playground.plutus.iohkdev.io/doc/haddock>.
```
These should be sufficient, no?
If so, should be actually ask for `plutus-playground-plutus-apps-staging.plutus.aws.iohkdev.io` to be shut down?

[the remark this answers is not in the document]
Comment (koslambrou) [reply README.adoc:172]: The `https://playground.plutus.iohkdev.io/doc/haddock` server hosts the haddock for a released version of plutus-apps. Whereas `plutus-playground-plutus-apps-staging.plutus.aws.iohkdev.io` hosts the haddock for the `main` branch. Both are useful. If you have a better way to restructure this, please suggest :)

**Comment by andreabedini** [comment]
image

This must be a new record :D

File: README.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (zeme-iohk) [reply README.adoc:174]: There are other places in the README that say where to find the docs, so I think it's safe to delete those lines.
As for a possible restructure: if it works, and both are really useful, I wouldn't change it :)

**Comment by Zaczero** [comment]
Hey, sorry for a newbie question, where should I search for the information on reasons for deleting the plutus-playground? Has it been moved somewhere else?

**Comment by baibinghere** [comment]
I have the same question. Is this community not explain anything now?

**Comment by eric248550** [comment]
@baibinghere 
https://github.com/input-output-hk/plutus-apps/issues/653#issuecomment-1326453253

Screen Shot 2022-12-28 at 6 33 33 PM

**Pull request merged**